engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/content/content-host-packages-installed.controller.js

```diff
@@ -18,7 +18,7 @@ angular.module('Bastion.content-hosts').controller('ContentHostPackagesInstalled
         var packagesNutupane;
 
         $scope.removeSelectedPackages = function () {
-            var selected = $scope.table.getSelected();
+            var selected = _.map($scope.table.getSelected(), 'name');
```

Comment:
The old implementation would take anything and everything from the table and pass it along to the API. The API docs for the package param indicate an array of strings which are package names - so making that consistent.